RELEASE.md

```diff
@@ -23,4 +23,11 @@ Keras pruning API:
 
  * Tested against TensorFlow 1.14.0, 2.0.0, and nightly.
 
-
+Keras clustering API:
+
+* New API for clustering of weights
+* Major Features
+  * Support for clustering of convolutional (except DepthwiseConv), fully connected and other commonly used standard Keras layers
```

Comment:
fully connected -> Dense to match Keras terminology.

Reply:
I will update this and also move the whole block to the top. Thanks for the feedback.